tenseal/tensors/bfvvector.cpp

```diff
@@ -190,16 +190,16 @@ BFVVector& BFVVector::mul_plain_inplace(const vector<int64_t>& to_mul) {
         throw invalid_argument("can't multiply vectors of different sizes");
     }
 
-    // TODO: rmeove this after fixing #36
-    // prevent transparent ciphertext by adding a non-zero value
     Plaintext plaintext;
-    vector<int64_t> new_vec_to_mul(to_mul);
-    if (new_vec_to_mul.size() + 1 <= this->context->slot_count<BatchEncoder>())
-        new_vec_to_mul.push_back(1);
-    this->context->encode<BatchEncoder>(new_vec_to_mul, plaintext);
-
-    this->context->evaluator->multiply_plain_inplace(this->ciphertext,
-                                                     plaintext);
+    this->context->encode<BatchEncoder>(to_mul, plaintext);
```

Comment:
SEAL evaluator's docs specify the transparent ct exception for more operations, like negate, add, relinearize etc. 

should we cover all?

Reply:
I'm still not sure what are the use case for those, so far using CKKS, multiplication has been the most important operation to deal with, so I would love to discover those failing use cases first before catching them :smile: